editors/emacs/swarm-mode.el

```diff
@@ -25,7 +25,30 @@
   "Syntax table for `swarm-mode'.")
 
 (defvar swarm-mode-operators-regexp
-  (regexp-opt '(":" "->" "=" "<-" "+" "*" "/" "-") t)
+  (regexp-opt
+   '(
+     "-"
+     "=="
+     "!="
+     "<"
+     ">"
+     "<="
+     ">="
+     "||"
+     "&&"
+     "+"
+     "-"
+     "*"
+     "/"
+     "\\"
+     ":"
+     "^"
+     "++"
+     "$"
+     "->"
+     "<-"
+     "=")
+   t)
```

Comment:
It seems that the list of operators are not in sync across editors. I'd like a second pair of eyes to check if this covers all or not.

Reply:
* What is `\\`? Is that lambda?
* There is duplicate `-` because one is unary, but that is OK.
* `++` will not matter I think, because `+` will be parsed twice instead. Should not be noticeable though.

It's probably best idea to test in emacs, but it looks good to me. 👍

Reply:
Why add it manually when you can copy the output of `cabal run swarm:swarm-docs -- editors --emacs`?

Reply:
I need to get some input on what the operators are for now. 
The list generated by `operatorNames` in it's current implementation does not cater to emacs (and possibly vim). Emacs fails to parse the generated list because of escaping where it does not need it.

But ultimately I'll be using that command once I figure out the requirements by emacs and vim.

Reply:
@xsebek My recent commit ensures that there's no manual labour involved. All the operators can now simply be copied to respective config files by running the `cabal run` command.